trace_log/src/main/java/gov/cdc/sdp/cbr/trace/DatabaseQueueComponent.java

```diff
@@ -0,0 +1,87 @@
+package gov.cdc.sdp.cbr.trace;
```

Comment:
Maybe I'm missing something, but I don't know why we have the DatabaseQueue stuff in here. Isn't all of this in the database_queue project?